notebooks/python_intro/03-containers.md

```diff
@@ -264,7 +264,7 @@ Select the last 3 elements of the list `pressures_hPa`.
 <details><summary>Hints</summary>
 
 - You can count elements backwards from the end by using a negative index.
-- The syntax to select elements is `L[start:stop:stride]`, with each of them optional.
+- The syntax to select elements is `L[start:stop:step]`, with each of them optional.
```

Comment:
Small note: the text was using both step or stride in different places. I change stride to step wherever I saw it (I think step is the most common term, and is also what you use in the prose text)